plugins/vagrant/vagrant.plugin.sh

```diff
@@ -0,0 +1,70 @@
+#! bash oh-my-bash.module
+# Functions
+function vagrant-version(){
+    vagrant --version
+}
+
+function vagrant-init(){
+    if [ -n "$1" ] ; then
+        echo "Vagrant init for : $1 Creating...."
+        vagrant init -m "$1"
+    else
+        echo "Usage : vai <box name>"
+        echo "Example : vai centos/7"
+    fi
+}
+
+function vagrant-up(){
+    if [ -n "$1" ] ; then
+        echo "Vagrant up for provider : $1 Running...."
+        vagrant up --provider "$1"
+    else
+        echo "Vagrant up for provider : virtualbox Running...."
+        vagrant up
+    fi
+}
+
+function vagrant-plugin-vm(){
+    case "$1" in
+        "virtualbox")
+            echo "Vagrant plugin install for provider : $1 Running...."
+            vagrant plugin install vagrant-vbguest
+        ;;
+        "libvirt")
+            echo "Vagrant plugin install for provider : $1 Running...."
+            vagrant plugin install vagrant-libvirt
+        ;;
+        *)
+            echo "Usage : vapvm <provider name>"
+            echo "Example : vapvm virtualbox"
+        ;;
+    esac
+}
+
+function vagrant-status(){
+    if [ -f "Vagrantfile" ]; then
+        vagrant status
+    else
+        vagrant global-status
+    fi
+}
+
+
+# Alias
+alias va='vagrant'
+alias vaver='vagrant-version'
+alias vaconf='vagrant ssh-config'
+alias vastat='vagrant-status'
+alias vacheck='vagrant validate'
+alias vaport='vagrant port'
+alias vapvm='vagrant-plugin-vm'
+alias vapi='vagrant plugin install'
+alias vapr='vagrant plugin uninstall'
+alias vau='vagrant-up '
```

Comment:
Is this space after `vagrant-up` intentional? In Bash, aliases that end with space have a special meaning that the first argument should also become subject to the alias expansions. However, as far as I search, `vagrant-up` doesn't seem to receive commands as arguments or some other types of arguments that can be usefully expanded by aliases.